library/src/main/java/com/alamkanak/weekview/WeekViewEvent.java

```diff
@@ -113,10 +126,20 @@ public WeekViewEvent(long id, String name, String location, Calendar startTime,
      * @param startTime The time when the event starts.
      * @param endTime The time when the event ends.
      */
-    public WeekViewEvent(long id, String name, Calendar startTime, Calendar endTime) {
+    public WeekViewEvent(String id, String name, Calendar startTime, Calendar endTime) {
         this(id, name, null, startTime, endTime);
     }
 
+    /**
+     * Initializes the event for week view.
+     * @param id The id of the event.
+     * @param name Name of the event.
+     * @param startTime The time when the event starts.
+     * @param endTime The time when the event ends.
+     */
+    public WeekViewEvent(int id, String name, Calendar startTime, Calendar endTime) {
```

Comment:
Why did you add this one specifically ? I don't see the logic behind this.

Reply:
These are overrides for the ```BaseActivity``` in the sample. I guess the helpers can move to the sample app, and dont need to be here.